frontend/components/draft/metadata/clickable-badges.tsx

```
@@ -0,0 +1,434 @@
+  selectedTags: Tag[];
+  availableTags: Tag[];
+};
+
+const focusAreaOptions = ["Personal", "Professional", "Academic"];
+const typeOptions = ["Skill", "Knowledge"];
+
+export function ClickableBadges({
+  focusArea,
+  type,
+  dropletId,
+  tags: initialTags,
```

ESLint

[error] 'initialTags' is defined but never used.

(@typescript-eslint/no-unused-vars)